index.html

```diff
@@ -11,14 +11,14 @@
     
     {% include pod_leader.html %}
 
-    {% include members.html %}
+  <!-- {% include members.html %} -->  
```

Comment:
Why was the members.html changed @amalaabraham @suhanichawla any idea I just noticed it now

Reply:
Oops! Uncommented and made a PR #28